[2021-03-25 14:54:24,719] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: EDGAR-Inference-Pipeline.Prediction 2021-03-25T21:54:02.481651+00:00 [queued]>
[2021-03-25 14:54:24,728] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: EDGAR-Inference-Pipeline.Prediction 2021-03-25T21:54:02.481651+00:00 [queued]>
[2021-03-25 14:54:24,729] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-25 14:54:24,729] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-25 14:54:24,729] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-25 14:54:24,737] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Prediction> on 2021-03-25T21:54:02.481651+00:00
[2021-03-25 14:54:24,742] {standard_task_runner.py:52} INFO - Started process 4868 to run task
[2021-03-25 14:54:24,749] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'EDGAR-Inference-Pipeline', 'Prediction', '2021-03-25T21:54:02.481651+00:00', '--job-id', '36', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/edgar_inference.py', '--cfg-path', '/var/folders/_j/s8vbc0cs1fvgclq8hnvtq1wm0000gn/T/tmprm998tt9', '--error-file', '/var/folders/_j/s8vbc0cs1fvgclq8hnvtq1wm0000gn/T/tmp85j7s13t']
[2021-03-25 14:54:24,751] {standard_task_runner.py:77} INFO - Job 36: Subtask Prediction
[2021-03-25 14:54:24,797] {logging_mixin.py:104} INFO - Running <TaskInstance: EDGAR-Inference-Pipeline.Prediction 2021-03-25T21:54:02.481651+00:00 [running]> on host prathyushas-mbp.home
[2021-03-25 14:54:24,832] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=EDGAR-Inference-Pipeline
AIRFLOW_CTX_TASK_ID=Prediction
AIRFLOW_CTX_EXECUTION_DATE=2021-03-25T21:54:02.481651+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-25T21:54:02.481651+00:00
[2021-03-25 14:55:16,567] {python.py:118} INFO - Done. Returned value was: None
[2021-03-25 14:55:16,572] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=EDGAR-Inference-Pipeline, task_id=Prediction, execution_date=20210325T215402, start_date=20210325T215424, end_date=20210325T215516
[2021-03-25 14:55:16,589] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-25 14:55:16,622] {local_task_job.py:146} INFO - Task exited with return code 0
